lib/ProductOpener/Food.pm

```diff
@@ -399,8 +399,14 @@ sub unit_to_g($$) {
 	$unit eq 'fl oz' and return $value * 30;
 
 	# return value without modification if it's already grams or 克 (kè) or 公克 (gōngkè) or г
-	return $value + 0; # + 0 to make sure the value is treated as number
+	# We return with + 0 to make sure the value is treated as number
 	# (needed when outputting json and to store in mongodb as a number)
+	(($unit eq 'g') or ($unit eq '克') or ($unit eq '公克') or ($unit eq 'г')
+		or ($unit eq 'мл') or ($unit eq '\N{U+6BEB}\N{U+5347}') or ($unit eq 'ml')
```

Comment:
```suggestion
		or ($unit eq 'мл') or ($unit eq "\N{U+6BEB}\N{U+5347}") or ($unit eq 'ml')
```

Strings within single quotes will not convert unicode characters sequence to the corresponding character, so we need to use double quotes here.

Reply:
Thanks for catching! I'll add some testcase so we can catch this.
/todo